### PR TITLE
Add openstreetmap as basemap.

### PR DESCRIPTION
**Pull request by na9da**
### What this PR does

Adds openstreetmap as a basemap.
Sticking with current setup, this would still require change to simple.json in terriamap to use as default basemap.

### Test me

- Open ci for this branch - http://ci.terria.io/add-openstreetmap
- From map settings select openstreet map as basemap

### Checklist

- [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
- [ ] I've updated relevant documentation in `doc/`.
- [x] I've updated CHANGES.md with what I changed.
- [ ] I've provided instructions in the PR description on how to test this PR.

### CODE REVIEW

**Review by nf-s — APPROVED** [review]
Awesome, thanks!

Just one small thing, can you please make the thumbnail a square so it matches the others

image

Thanks!

**Comment by na9da** [comment]
> Awesome, thanks!
> 
> Just one small thing, can you please make the thumbnail a square so it matches the others
> image
> 
> Thanks!

Yep, also I need to reduce the size of thumbnail.

File: lib/Models/BaseMaps/defaultBaseMaps.ts [hunk 1]
```diff
@@ -94,5 +94,19 @@ export function defaultBaseMaps(terria: Terria): BaseMapJson[] {
...
+    item: {
+      id: "basemap-openstreetmap",
+      name: "OpenStreetMap",
+      type: "open-street-map",
+      url: "https://tile.openstreetmap.org/",
+      attribution:
+        "© <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap contributors</a>",
+      opacity: 1.0
+    },
+    image: "build/TerriaJS/images/openstreetmap.png",
```
Comment (pjonsson) [inline lib/Models/BaseMaps/defaultBaseMaps.ts:107]: Can this file be used in `test/Models/BaseMaps/BaseMapsModelSpec.ts` and `test/Models/TerriaSpec.ts` so `dark-matter.png` that belonged to one of the removed maps can be removed?

Comment (na9da) [reply lib/Models/BaseMaps/defaultBaseMaps.ts:107]: Thanks. I have removed references to these images from the specs for now.
We are not testing anything about the images at this point.

**Pull request merged**